dashboard/app/helpers/users_helper.rb

```diff
@@ -1,4 +1,106 @@
 module UsersHelper
+  include ApplicationHelper
+
+  # Returns the number of lines written in the current user session.
+  # return [Integer]
+  def session_lines
```

Comment:
when there are a bunch of methods that all start with the same string I am thinking that it wants to be in a namespace...